storage/src/epoch.rs

```diff
@@ -1,30 +1,31 @@
-use cardano::block::{BlockDate, ChainState, EpochId};
+use cardano::block::{types::HeaderHash, BlockDate, ChainState, EpochFlags, EpochId};
 use cardano::config::GenesisData;
-use cardano::util::hex;
 use chain_state;
 use std::fs;
-use std::io::Read;
+use std::io::{Read, Seek, SeekFrom, Write};
 
 use super::{
     header_to_blockhash, packreader_block_next, packreader_init, Error, PackHash, Result, Storage,
     StorageConfig,
 };
 use storage_units::utils::error::StorageError;
-use storage_units::utils::tmpfile;
 use storage_units::utils::tmpfile::TmpFile;
-use storage_units::{packfile, reffile};
+use storage_units::utils::{serialize, tmpfile};
+use storage_units::{hash, indexfile, packfile, reffile};
 
 pub fn epoch_create_with_refpack(
     config: &StorageConfig,
     packref: &PackHash,
     refpack: &reffile::Lookup,
     epochid: EpochId,
+    index: indexfile::Index,
+    flags: &EpochFlags,
 ) {
     let dir = config.get_epoch_dir(epochid);
     fs::create_dir_all(dir).unwrap();
 
-    let pack_filepath = config.get_epoch_pack_filepath(epochid);
-    tmpfile::atomic_write_simple(&pack_filepath, hex::encode(packref).as_bytes()).unwrap();
+    epoch_write_pack(config, packref, &None, epochid, index.offsets, flags).unwrap();
+    // TODO: need to put new entry to storage, but storage is not present here =(
```

Comment:
⚠️ Idk what to do with this function, cuz by function definition we don't have all the context we need in here, but this function is never actually called anywhere, so we gotta either just delete it, or maybe add required context to function arguments, so if it's ever called - caller will be aware of it.